src/components/common/filters/ClearFilter.js

```diff
@@ -0,0 +1,21 @@
+import { Box, Button} from '@chakra-ui/react'
+import "../../../styles/search-style.css";
+
+const ClearFilter = ({ onClick}) => {
+    return (
+        <Box sx={{ display: 'flex', flexDirection: 'row', justifyContent: 'flex-start' }}>
+            <Button
+                variant="link"
+                onClick={onClick}
+                className="search-reset"
+                _hover={{ textDecor: "none" }}
+            >
+                Clear Filter
```

Comment:
I would relabel to "Clear filters", I didn't realize it clears all the filters, not just the one right above the clear filter button